reactimail/message_template/forms.py

```
@@ -0,0 +1,32 @@
+
+    def clean_body(self):
+        body = self.cleaned_data.get("body")
+
+        # Syntax checking and parameter extraction.
+        try:
+            tmpl = Template(body)
+            parameters = pattern.findall(tmpl.template)
+            # For checking the parameter list
+            self.cleaned_data["parameters"] = parameters
+        except ValueError as e:
+            raise ValidationError(f"Template syntax error: {e}")
```

Ruff

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)